vendor/github.com/docker/distribution/testutil/handler.go

```diff
@@ -0,0 +1,148 @@
+package testutil
```

Comment:
In the end, the dependency wasn't really necessary. The imported function `CreateRandomTarFile` created unnecessarily large one file taking too much time.